crates/bevy_xpbd_2d/examples/one_way_platform_2d.rs

```diff
@@ -0,0 +1,305 @@
+    ));
+    // Floor
+    commands.spawn((
+        SpriteBundle {
+            sprite: square_sprite.clone(),
+            transform: Transform::from_scale(Vec3::new(20.0, 1.0, 1.0)),
+            ..default()
+        },
+        RigidBody::Static,
+        Position(Vector::NEG_Y * 50.0 * 6.0),
+        Collider::cuboid(50.0 * 20.0, 50.0),
+    ));
+    // Left wall
+    commands.spawn((
+        SpriteBundle {
+            sprite: square_sprite.clone(),
+            transform: Transform::from_scale(Vec3::new(1.0, 11.0, 1.0)),
+            ..default()
+        },
+        RigidBody::Static,
+        Position(Vector::NEG_X * 50.0 * 9.5),
+        Collider::cuboid(50.0, 50.0 * 11.0),
+    ));
+    // Right wall
+    commands.spawn((
+        SpriteBundle {
+            sprite: square_sprite,
+            transform: Transform::from_scale(Vec3::new(1.0, 11.0, 1.0)),
+            ..default()
+        },
+        RigidBody::Static,
+        Position(Vector::X * 50.0 * 9.5),
+        Collider::cuboid(50.0, 50.0 * 11.0),
+    ));
+
+    // For one-way platforms
+    let one_way_sprite = Sprite {
+        color: Color::rgba(0.7, 0.7, 0.8, 0.25),
+        custom_size: Some(Vec2::splat(50.0)),
+        ..default()
+    };
+
+    // Spawn some one way platforms
+    for y in -2..=2 {
+        commands.spawn((
+            SpriteBundle {
+                sprite: one_way_sprite.clone(),
+                transform: Transform::from_scale(Vec3::new(10.0, 0.5, 1.0)),
+                ..default()
+            },
+            RigidBody::Static,
+            Position(Vector::Y * 15.0 * 6.0 * y as Scalar),
+            Collider::cuboid(25.0 * 20.0, 25.0),
+            OneWayPlatform::default(),
+        ));
+    }
+
+    // Spawn an actor for the user to control
+    let actor_size = Vec2::new(20.0, 20.0);
+    let actor_mesh = MaterialMesh2dBundle {
+        mesh: meshes.add(shape::Quad::new(actor_size).into()).into(),
+        material: materials.add(ColorMaterial::from(Color::rgb(0.2, 0.7, 0.9))),
+        ..default()
+    };
+
+    commands.spawn((
+        actor_mesh.clone(),
+        RigidBody::Dynamic,
+        LockedAxes::ROTATION_LOCKED,
+        Position(Vector::ZERO),
+        Collider::cuboid(actor_size.x.into(), actor_size.y.into()),
+        Actor,
+        PassThroughOneWayPlatform::ByNormal,
+    ));
+}
+
+fn movement(
+    keyboard_input: Res<Input<KeyCode>>,
+    mut actors: Query<&mut LinearVelocity, With<Actor>>,
+) {
+    for mut linear_velocity in &mut actors {
+        if keyboard_input.pressed(KeyCode::Left) {
+            linear_velocity.x -= 10.0;
+        }
+        if keyboard_input.pressed(KeyCode::Right) {
+            linear_velocity.x += 10.0;
+        }
+
+        if linear_velocity.y.abs() < 0.1 // Assume "mostly stopped" to mean "grounded"
+            && !keyboard_input.pressed(KeyCode::Down)
+            && keyboard_input.just_pressed(KeyCode::Space)
+        {
+            linear_velocity.y = 500.0;
```

Comment:
The jump height feels just a bit too large in my opinion; maybe reduce this to 450.0 or increase the gravity a bit?
```suggestion
            linear_velocity.y = 450.0;
```